conda_smithy/feedstock_content/build-locally.py

```diff
@@ -26,6 +26,12 @@ def setup_environment(ns):
             os.path.dirname(__file__), "miniforge3"
         )
 
+    # The default cache location might not be writable using docker on macOS.
+    if ns.config.startswith("linux") and platform.system() == "Darwin":
+        os.environ["CONDA_FORGE_DOCKER_RUN_ARGS"] = (
+            "-e XDG_CACHE_HOME=/tmp/rattler_cache"
+        )
```

Comment:
Let's make sure we are not overwriting a maybe-present CONDA_FORGE_DOCKER_RUN_ARGS. Something like:

```suggestion
    if ns.config.startswith("linux") and platform.system() == "Darwin":
        os.environ["CONDA_FORGE_DOCKER_RUN_ARGS"] = (
            os.environ.get("CONDA_FORGE_DOCKER_RUN_ARGS", "")
            + " -e RATTLER_CACHE_DIR=/tmp/rattler_cache"
        )
```

Reply:
Yes good catch. Thanks.